dist/modules/sound.js

```diff
@@ -691,13 +698,21 @@
 			}
 			//if the sound plugin is not ready, then just wait until it is
 			var waitFunction;
+			var waitResult;
+
 			waitFunction = function()
 			{
-				if (SoundJS.getCapabilities())
+				var NativeAudio = window.plugins.NativeAudio || window.top.plugins.NativeAudio || null;
+				if (NativeAudio)
 				{
-					Application.instance.off("update", waitFunction);
-					_instance._initComplete(options.types, options.ready);
+					NativeAudio.getCapabilities(function(result)
+					{
+						waitResult = result;
+					});
 				}
+
+				Application.instance.off("update", waitFunction);
```

Comment:
Yes, you're right my bad. I think I intended that to be in call back block but moved it out. Unless I did that intentionally...